cudax/test/stream/get_stream.cu

```diff
@@ -69,7 +69,67 @@ struct get_stream_wrong_return
   }
 };
 TEST_CASE("The get_stream method must return a cuda::stream_ref", "[stream]")
+{
+  static_assert(!::cuda::std::is_invocable_v<::cuda::experimental::get_stream_t, const get_stream_wrong_return&>);
```

Comment:
This is a Catch2 test, right? Because then there is the `STATIC_REQUIRE` macro for compile-time test.
```suggestion
  STATIC_REQUIRE(!::cuda::std::is_invocable_v<::cuda::experimental::get_stream_t, const get_stream_wrong_return&>);
```

Reply:
We can bit I never really understood the advantage of it

Reply:
From their docs:

> STATIC_REQUIRE( expr ) is a macro that can be used the same way as a static_assert, but also registers the success with Catch2, so it is reported as a success at runtime. 

So running the test actually shows that it checked something (even if the checking was done at compile time).